operator/gitops/argocd/pipeline-service/openshift-pipelines/chains-secrets-config.yaml

```
@@ -61,37 +61,43 @@ spec:
             - /bin/bash
             - -c
             - |
+              set -o errexit
+              set -o nounset
+              set -o pipefail
+
+              namespace="openshift-pipelines"
+              secret="signing-secrets"
+
               cd /tmp
-              # Try to handle that nicely. The object is expected to always exist so check the data.
-              SIG_KEY_DATA=$(kubectl get secret signing-secrets -n openshift-pipelines -o jsonpath='{.data}')
-              if [[ -n $SIG_KEY_DATA ]]; then
+
+              if [ "$(kubectl get secret "$secret" -n "$namespace" -o jsonpath='{.data}')" != "" ]; then
                 echo "Signing secret exists and is non-empty."
               else
                 # Delete secret/signing-secrets if already exists since by default cosign creates immutable secrets
-                kubectl delete secrets -n openshift-pipelines signing-secrets --ignore-not-found=true
+                kubectl delete secrets "$secret" -n "$namespace" --ignore-not-found=true
 
                 # To make this run conveniently without user input let's create a random password
                 RANDOM_PASS=$( head -c 12 /dev/urandom | base64 )
```

A bit stronger passphrase and uses single command

```suggestion
                RANDOM_PASS=$( openssl rand -base64 30 )
```

---

Done